src/clients/Device.js

```diff
@@ -811,15 +813,13 @@ class Device extends EventEmitter {
   };
 
   releaseOwnership = (flasher: Flasher) => {
-    logger.log('releasing flash ownership ', { deviceID: this.getDeviceID() });
+    logger.info({ deviceID: this.getDeviceID() }, 'releasing flash ownership ');
     if (this._owningFlasher === flasher) {
       this._owningFlasher = null;
     } else if (this._owningFlasher) {
       logger.error(
-        'cannot releaseOwnership, ',
-        flasher,
-        " isn't the current owner ",
-        { deviceID: this.getDeviceID() },
+        { deviceID: this.getDeviceID(), flasher },
+        'cannot releaseOwnership, isn\'t  current owner'
```

Comment:
Hey, sorry if I was rude, but when some lines before I see commented lint-staged task, I can't react another way :). Don't go crazy about that commas, we'll figure it out. I thought that `prettier --trailing-comma all` should add all the commas. Anyways, big thanks for your contributions.